modules/statics/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bitgo/statics",
-  "version": "6.12.0",
+  "version": "6.13.0-rc.9",
```

Comment:
I've updated the version here because of the check-versions script running on the drone